### ISSUE TITLE
Modify the artifact manager to receive magnet links and use them to receive artifacts.

### ISSUE THREAD

**Issue by mgrand** [pyrsia/pyrsia#279]
This is to be done after #278

This task is to modify artifact manager to receive magnet link from a pubsub mechanism and use them with Synapse to download artifacts. This processing of magnet links should happen asynchronously of any incoming warp requests.

**Comment by efrisch** [comment]
Closing due to bittorrent stories being out of scope / replaced with bitswap.
